docs/create-osiam-connector.md

```diff
@@ -2,17 +2,55 @@ To be able to log in, and create or change users or groups you need to create
 an `org.osiam.client.connector.OsiamConnector` instance. You can do this by
 using the `OsiamConnector.Builder()` class.
 
+## OSIAM 3.x
+
+In OSIAM 3.x auth-server and resource-server have been merged to create a better
+user experience. Thus you have to provide only 1 endpoint for the Connector to
```

Comment:
The docs must be changed anyways, when OSIAM 3.x is out for a while and support for 2.x will be dropped or deprecated.